extensions/dicom-microscopy/src/DicomMicroscopyViewport.tsx

```diff
@@ -1,318 +1,217 @@
-    const loadViewer = async metadata => {
-      const dicomMicroscopyModule = await this.microscopyService.importDicomMicroscopyViewer();
-      const { viewer: DicomMicroscopyViewer, metadata: metadataUtils } = dicomMicroscopyModule;
+  const installOpenLayersRenderer = useCallback(
+    async (container, displaySet) => {
+      const loadViewer = async metadata => {
+        const dicomMicroscopyModule = await microscopyService.importDicomMicroscopyViewer();
+        const { viewer: DicomMicroscopyViewer, metadata: metadataUtils } = dicomMicroscopyModule;
 
-      const microscopyViewer = DicomMicroscopyViewer.VolumeImageViewer;
+        const microscopyViewer = DicomMicroscopyViewer.VolumeImageViewer;
 
-      const client = getDicomWebClient({
-        extensionManager: this.props.extensionManager,
-        servicesManager: this.props.servicesManager,
-      });
-
-      // Parse, format, and filter metadata
-      const volumeImages: any[] = [];
-
-      /**
-       * This block of code is the original way of loading DICOM into dicom-microscopy-viewer
-       * as in their documentation.
-       * But we have the metadata already loaded by our loaders.
-       * As the metadata for microscopy DIOM files tends to be big and we don't
-       * want to double load it, below we have the mechanism to reconstruct the
-       * DICOM JSON structure (denaturalized) from naturalized metadata.
-       * (NOTE: Our loaders cache only naturalized metadata, not the denaturalized.)
-       */
-      // {
-      //   const retrieveOptions = {
-      //     studyInstanceUID: metadata[0].StudyInstanceUID,
-      //     seriesInstanceUID: metadata[0].SeriesInstanceUID,
-      //   };
-      //   metadata = await client.retrieveSeriesMetadata(retrieveOptions);
-      //   // Parse, format, and filter metadata
-      //   metadata.forEach(m => {
-      //     if (
-      //       volumeImages.length > 0 &&
-      //       m['00200052'].Value[0] != volumeImages[0].FrameOfReferenceUID
-      //     ) {
-      //       console.warn(
-      //         'Expected FrameOfReferenceUID of difference instances within a series to be the same, found multiple different values',
-      //         m['00200052'].Value[0]
-      //       );
-      //       m['00200052'].Value[0] = volumeImages[0].FrameOfReferenceUID;
-      //     }
-      //     NOTE: depending on different data source, image.ImageType sometimes
-      //     is a string, not a string array.
-      //     m['00080008'] = transformImageTypeUnnaturalized(m['00080008']);
-
-      //     const image = new metadataUtils.VLWholeSlideMicroscopyImage({
-      //       metadata: m,
-      //     });
-      //     const imageFlavor = image.ImageType[2];
-      //     if (imageFlavor === 'VOLUME' || imageFlavor === 'THUMBNAIL') {
-      //       volumeImages.push(image);
-      //     }
-      //   });
-      // }
+        const client = getDicomWebClient({
+          extensionManager: extensionManager,
```

Comment:
Use use default assignment here please - eg, getDicomWebClient({extensionManager, servicesManager}) or even easier, just record the useSystem result and pass in system.